webapp/channels/src/components/admin_console/push_settings.jsx

```diff
@@ -262,6 +289,30 @@ export default class PushSettings extends AdminSettings {
             );
         }
 
+        let locationDropdown;
+        if (this.state.pushNotificationServerType === PUSH_NOTIFICATIONS_MHPNS) {
+            const pushNotificationServerLocations = [];
```

Comment:
you can declare the array with the values directly, like `const arr = [{a: 'a'}, {b: 'b'}]`, no need to use the `Array.push` method.